dbt/adapters/base/impl.py

```diff
@@ -782,21 +783,15 @@ def valid_snapshot_target(self, relation: BaseRelation) -> None:
 
         columns = self.get_columns_in_relation(relation)
         names = set(c.name.lower() for c in columns)
-        expanded_keys = ("scd_id", "valid_from", "valid_to")
-        extra = []
+        columns = ("dbt_scd_id", "dbt_valid_from", "dbt_valid_to")
```

Comment:
Redeclaring a variable can be confusing. Could we change the name of one of these? Or, given how short this is, maybe we just do it inline on 788:
```python
for column in ("dbt_scd_id", "dbt_valid_from", "dbt_valid_to"):
```

Reply:
This has been changed.